app/assets/stylesheets/semantic-ui/elements/_input.scss

```diff
@@ -461,9 +461,30 @@
 
 /* Standard */
 .ui.inverted.input input {
+  background: $white;
+  box-shadow: 0px 0px 0px 1px rgba(0,0,0, 0.2) inset !important;
```

Comment:
This box-shadow will also affect the fields with `.ui.inverted.transparent.input`